### PR TITLE
added credit in footer

### LINKED ISSUE #249: [HomePage] Add credits for the designer
### Description

The website was designed by @ericliclair. Add the credits in the footer of the website.

### Screenshots/testimonials

NA

### PR DESCRIPTION
**Pull request by ashwani3011**
### Fixes #249
<Do not delete this template>
Description:  I have added the credits in the footer of the website for website developer. Please have a look and please inform if change needed.

#### Checklist
- [✔️] I've followed the Contributing guidelines provided in the repository.
- [✔️] I've made the changes which were demanded in the linked issue.
- [✔️] I've tested my code on a Chromium based browser.
- [✔️] I've tested my code on Mozilla Firefox.
- [✔️] My code gave a clean console on debugging. (no warnings/errors)

#### Testimonial
image

### CODE REVIEW

File: client/src/components/Footer/Footer.js [hunk 1]
```diff
@@ -85,6 +85,14 @@ function Footer() {
...
               and open-sourced
             </p>
+
+            <p>
+              The website was designed by{" "}
+              <a className="credit" href="https://github.com/EricLiclair">
+                {" "}
+                @EricLiclair.
+              </a>
+            </p>
```
Comment (tusharjain0022) [inline client/src/components/Footer/Footer.js]: @ashwani3011, thanks for your PR, can you change the text to - `This` website `is` designed by @EricLiclair.

**Comment by ashwani3011** [comment]
I have made the suggested change kindly have a look

**Review by tusharjain0022 — APPROVED** [review]
Looks good to me :rocket: .

**Comment by tusharjain0022** [comment]
Great work @ashwani3011  ! Looking forward to seeing more contributions from you. 🙌🏻
Also, don't forget to star this repo if you think this is a great project. :star:

**Comment by tusharjain0022** [comment]
@EmperorYP7, can you clear the cache and re-built the code on Netlify, the changes of the last two PRs are not visible in production.

**Comment by tusharjain0022** [comment]
Edit:
@all-contributors please add @ashwani3011  for code

**Pull request merged**